script.js

```diff
@@ -1,4 +1,159 @@
+/* Two-player game
+Player 1 will press submit and 2 dice are rolled
+Player 1 to select dice order
+Player 2 will press submit and 2 dice are rolled
+Player 2 to select dice order
+To determine who the winner is */
+
+var playerOneChoice = "";
+var playerOneDiceRoll = "";
+var playerOneDiceOne = "";
+var playerOneDiceTwo = "";
+
+var playerTwoChoice = "";
+var playerTwoDiceRoll = "";
+var playerTwoDiceOne = "";
+var playerTwoDiceTwo = "";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  // refresh all variables to play again
+  if (playerTwoChoice !== "") {
+    playerOneChoice = "";
+    playerOneDiceRoll = "";
+    playerOneDiceOne = "";
+    playerOneDiceTwo = "";
+
+    playerTwoChoice = "";
+    playerTwoDiceRoll = "";
+    playerTwoDiceOne = "";
+    playerTwoDiceTwo = "";
+  }
```

Comment:
one common theme i have in my comments will be that your main function is very lengthy and should be refactored into further helper functions

this is one place that it can be done, consider having a resetGame function